### PR TITLE
Fix metadata json writeout

### PR DESCRIPTION
**Pull request by dpark6060**
metadata.json used to include full paths to file names ('/flywheel/v0/output/file.txt') which was causing upload errors and the gear would fail.

### CODE REVIEW

File: utils/Common.py [hunk 1]
```diff
@@ -55,3 +56,10 @@ def exec_command(context, command, shell=False, stdout_msg=None):
             raise Exception(stderr)
 
 
+def set_metadata(context, file, fdict):
+    # In the event that there is a full directory in front of the file name, remove it.
```
Comment (neurosutton) [inline utils/Common.py:60]: Removed by `.name`? Just checking that the directory doesn't need to get added back in l64, when file2 is sent to the dictionary...

File: run.py [hunk 2]
```diff
@@ -11,6 +11,7 @@
 from pprint import pprint as pp
 import copy
 import utils.physio as phys
+from pathlib import Path
```
Comment (neurosutton) [inline run.py:15]: nice import name choices...

**Review by neurosutton — APPROVED** [review]
LGTM. This ran okay (specifically with the change in stripping the dir name in Common.py?

File: utils/Common.py (hunk 1 above)
Comment (dpark6060) [reply utils/Common.py:60]: yes, it needed to be stripped in both places to work

Comment (dpark6060) [reply utils/Common.py:60]: which is done on line 44...this is a very old gear I would not lay it out this way anymore.  It deserves more of a facelift but maybe another sprint.

**Pull request merged**